### DISCUSSION TITLE
Why does YOLO achievement fire for your own work on single-person projects?

### DISCUSSION THREAD

**Discussion by cyrusae** [community/community#19138] [Other Feature Feedback, Questions, & Ideas]
What is the point of marking a pull request by the owner of a project with one contributor, merged by the same person, in a context where no competing pull requests exist, as "merged without review"? There's "it's possible to do code review offsite" but this just seems next-level. e.g., [this comment](https://github.com/github-community/community/discussions/18153#discussioncomment-2916150) demonstrates that users expect it to mean I did that to someone else. 

I realize there's a strong possibility people will just tell me to turn off badges, and I already did, but I did not *want* to--I would like to participate in the silly thing if it's here, having *this specific badge in context* is just asinine and leaves a bad taste in my mouth. (Especially with the insistent call-to-action to "update Profile settings" on my profile *after* manually turning them off, in Settings.)

**Comment by eldare** [comment]
I also got the YOLO for merging "without PR" on 1 person project.
The whole achievements thing on github is ridiculous. What's point of giving people the Pull Shark for 2 merges, for the one I made in 2018. The whole GitHub achievement thing is poorly executed at best.
